db/migrations/000013_add_index_to_posts_id_ref.up copy.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX posts_id_index ON posts (id);
```

Comment:
This file has `copy` in its name 😅 
+ I think this should not be necessary on the primary key